### ISSUE TITLE
Can't make hyperline taller or the font larger.

### ISSUE THREAD

**Issue by ChristosBouronikos** [Hyperline/hyperline#189]
I tried editing the .js file hidden in the local appdata, but still nothing changes.

**Comment by lastnamearya** [comment]
Yes, +1 :)

**Comment by rohitkrishna094** [comment]
any update on how to do this?

**Comment by ChristosBouronikos** [comment]
No. Not that I know of. I've searched through open issues and old closed issues. Nothing.
Are you on Windows 10 too?

**Comment by rohitkrishna094** [comment]
Yes I am windows 10.

**Comment by ChristosBouronikos** [comment]
Well the developer hasn't been active for months apparently, so I don't see this bug being fixed anytime soon.

**Comment by Sagarmak-zz** [comment]
Hey guys, I have my Hyper installed in directory `C:\Users\{username}\AppData\Roaming\Hyper` and inside of this, all the Hyper packages are stored in `.hyper_plugins` folder. Inside its same as a React or Vue project's `npm_modules` folder and inside of this, you can navigate to `hyperline` directory. In this folder, go to `dist/hyperline.js` and its a build file for the `hyperline` plugin. Go to line no. 197 or find `position:absolute;` to jump to a line which is the css of the hyperline plugin. Feel free to play around with the `font-size`, also can change the `height` of the bar. Do remember to reload the Hyper terminal if you want the changes to reflect. 

This is the actual file you'll be updating, [hyperline.js](https://github.com/Hyperline/hyperline/blob/master/src/lib/core/hyperline.js)

> Note: This is not an official way to change the css/design of the plugin, but as you're kinda asking it so here's a solution for you.

 Cheers :tada:

**Comment by ByFe** [comment]
I had a similar issue and just looked up the element class in the `npm_modules\hyperline\dist\hyperline.js` file and add some custom css in the `hyper.js` config file so that I didn't mess with the original plugin file. 
```` 
css: `
  .line.jsx-1196914242 {
    font:normal 12px Fira Code;
  }
`
````

**Comment by Sagarmak-zz** [comment]
This works, and its awesome. Thanks @ByFe ✌️

**Comment by mS0gxokXEsg5ysJ7J79kzWiqO1fHgFA6ZR8eEls** [comment]
css: ['.line.jsx-1196914242{font-size: 21px;font-family: Prima Sans Mono;transform: translateY(-3px);display:flex;align-items:center;position:absolute;overflow:hidden;bottom:0;width:100%;height:22px;font:bold 10px Monospace;pointer-events:none;background:rgba(0,0,0,0.08);margin:2px 0;padding:0 10px;}', '.jsx-1196914242,.wrapper.jsx-1196914242{font-size: 21px;font-family: Prima Sans Mono;transform: translateY(-3px);display:flex;flex-shrink:0;align-items:center;padding-left:10px;padding-right:10px;}']
